meta-balena-common/recipes-core/fatrw/fatrw_git.bb

```diff
@@ -1,6 +1,6 @@
 # Auto-Generated by cargo-bitbake 0.3.15
 #
-inherit cargo
+inherit cargo binary-compress
```

Comment:
We may drop UPX as we discussed it affects negatively performance. Also since the file is auto-generated probably we should not modify it manually as the changes could be lost on next version upgrade of fatrw.

Reply:
Yes, I removed it.